### PR TITLE
Code refrctor/remove duplicate selectors

### PR DESCRIPTION
**Pull request by skumarqa2010**
Removed the duplicate selectors.

### CODE REVIEW

File: cypress/integration/teacher-training-adviser.spec.js [hunk 1]
```diff
@@ -2417,8 +2417,8 @@ describe('Feature - Get an adviser : Tests execution date and time : ' + new Dat
 		cy.selectWhichClassIsYourDegree('First class');
 		cy.selectStage('Secondary');
 
-		cy.doYouHaveGrade4CorAboveInEnglishAndMathsGCSEsorEuivalent('No', 'Secondary');
-		cy.areYouPlanningToRetakeEitherEnglishorMathsorBothGCSEsorEquivalent('Yes', 'Secondary');
+		cy.doYouHaveGrade4CorAboveInEnglishAndMathsGCSEsorEuivalent('No');
```
Comment (ethax-ross) [inline cypress/integration/teacher-training-adviser.spec.js:2420]: Typo in this method name. I'd be inclined to simplify it a bit to something like:

```
cy.gcseMathsAndEnglish(false);
```

File: cypress/support/commands.js [hunk 2]
```diff
@@ -369,28 +369,15 @@ Cypress.Commands.add('whichSubjectAreYouInterestedInTeaching', (subject) => {
...
-				cy.get(
-					'#teacher-training-adviser-steps-gcse-maths-english-has-gcse-maths-and-english-id-222750000-field'
-				).click();
-			} else {
-				cy.get(
-					'#teacher-training-adviser-steps-gcse-maths-english-has-gcse-maths-and-english-id-222750001-field'
-				).click();
-			}
+	(haveGrade) => {
+		if(haveGrade=="No")
```
Comment (ethax-ross) [inline cypress/support/commands.js:373]: Can `haveGrade` be a boolean?

**Review by ethax-ross — COMMENTED** [review]
Much simpler, just a couple of comments

File: cypress/integration/teacher-training-adviser.spec.js (hunk 1 above)
Comment (skumarqa2010) [reply cypress/integration/teacher-training-adviser.spec.js]: Resolved

File: cypress/support/commands.js (hunk 2 above)
Comment (skumarqa2010) [reply cypress/support/commands.js]: Implemented

**Review by ethax-ross — APPROVED** [review]
Looks good 👍

**Pull request merged**